### ISSUE TITLE
autogen.sh: No such file or directory

### ISSUE THREAD

**Issue by codenotworking** [mono/taglib-sharp#188]
New to Linux.  I installed Ubuntu 19.10.  Ran the following commands from the home page:

```
git clone https://github.com/mono/taglib-sharp.git
cd taglib-sharp
./autogen.sh && make
```

What am I doing wrong?
Any help appreciated.

**Comment by decriptor** [comment]
Sorry, I made a fair change and forgot to update the README.md. There is no longer the autogen.sh stuff. I'm not at a linux machine right now, but this should probably work.

`msbuild /restore TaglibSharp.sln`

How are you trying to use taglib-sharp? Are you just consuming the dll? or trying to include it in your own solution? If you can use a nuget, I'd recommend this: https://www.nuget.org/packages/TagLibSharp/

**Comment by codenotworking** [comment]
Thanks for the advice.  I installed Ubuntu on a virtual machine.  I had to install [.NET Core SDK for Linux/Ubuntu](https://dotnet.microsoft.com/download).  Then I ran the command you mentioned and successfully built the DLL file.  But I noticed there were error messages about not having .NET Framework 4.5 or later installed.

I instead decided to install the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/?q=build+tools), which has **MSBUILD.exe**, on my Windows 10 machine instead.  It actually was far more simpler for me on Windows, than on Ubuntu.  The cool thing is it built both .NET Framework 2.0 and 4.5 versions of the DLL.

My main purpose was just building the DLL file so I can utilize it in PowerShell, and automating the script with AutoHotKey, for renaming MP3 tags.
